README.md

```diff
@@ -82,6 +82,18 @@ A **Vagrant** configuration is included for development purposes. To use it, com
 - Run `vagrant ssh -c "cd /vagrant && foreman start"`
 - Open `http://mastodon.local` in your browser
 
+To set up **M1/2 MacOS** for native development, complete the following steps:
+
+- Install Ruby 3.2 (using [rbenv](https://github.com/rbenv/rbenv) for easy installation and management of ruby versions)
+- Run `brew install postgresql@15`
+- Run `brew install redis`
+- Run `brew install imagemagick`
+- Install Foreman or a similar tool (such as [overmind](https://github.com/DarthSim/overmind)) to handle multiple process launching.
+- Navigate to Mastodon's root directory and run `brew install nvm` then `nvm use` (Node 16.20.1 at time of writing)
```

Comment:
Might omit the 16.20.1 here and just say "latest stable" or "the version from nvmrc"